specification/network/resource-manager/Microsoft.Network/stable/2018-04-01/virtualWan.json

```diff
@@ -0,0 +1,2125 @@
+{
```

Comment:
@MikhailTryakhov Was this spec reviewed?

Reply:
@dsgouda  https://github.com/Azure/azure-rest-api-specs/pull/3204